jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/saved/SavedFiltersPresenter.java

```diff
@@ -105,4 +113,15 @@ public void onRestoreDefaultFilters() {
                                                                                  Commands.DO_NOTHING);
         yesNoCancelPopup.show();
     }
+
+    public void onSavedFilterAsDefaultActive(@Observes final SavedFilterAsDefaultActiveEvent event) {
+        YesNoCancelPopup yesNoCancelPopup = YesNoCancelPopup.newYesNoCancelPopup(constants.SavedFilterAsDefaultTitle(),
```

Comment:
I'm not sure we actually need a confirmation popup in here, just to select a filter as default. It doesnt sound much of a problem if you do it by mistake, wdyt? @jstastny-cz ?

Reply:
I agree in this case it is not necessary, given the user can undo easily.